packages/core/helper-plugin/lib/src/components/GenericInput/tests/index.test.js

```diff
@@ -128,4 +150,39 @@ describe('GenericInput', () => {
       expect(container).toMatchSnapshot();
     });
   });
+
+  describe('datetime', () => {
+    test('renders the datetime picker with the correct value for date and time', async () => {
+      const user = userEvent.setup();
+      const { getByRole } = setupDatetimePicker();
+      const btnDate = getByRole('textbox', { name: /datetime/i });
+      await user.click(btnDate);
+      await user.click(getByRole('button', { name: /15/ }));
+      const today = new Date();
+      const month = today.getMonth() + 1;
+      const year = today.getFullYear();
+      
+      expect(getByRole('textbox', { name: 'datetime' })).toHaveValue(`${month}/15/${year}`);
+      expect(getByRole('combobox', { name: /datetime/i })).toHaveValue('00:00');
+    });
+  });
+
+  test('simulate clicking on the Clear button in the date and check if the date and time are empty', async () => {
+      const user = userEvent.setup();
+      const { getByRole } = setupDatetimePicker();
+      const btnDate = getByRole('textbox', { name: /datetime/i });
+      await user.click(btnDate);
+      await user.click(getByRole('button', { name: /15/ }));
+      const today = new Date();
+      const month = today.getMonth() + 1;
+      const year = today.getFullYear();
+      
+      expect(getByRole('textbox', { name: 'datetime' })).toHaveValue(`${month}/15/${year}`);
+      expect(getByRole('combobox', { name: /datetime/i })).toHaveValue('00:00');
```

Comment:
This part is the same in both tests - should it be removed from the second as the second one focuses on canceling?

Reply:
You are right, I tried to pass directly the initial value to clear as an argument to the setupDatetimePicker but then, when I simulate the click on the clear date button the date remains the initial one not the empty one

Reply:
The setup part is fine, but you shouldn't run the same tests twice and at least remove the two duplicated `expect` calls.

Reply:
ok

Reply:
done